net-im/psi/metadata.xml

```diff
@@ -16,7 +16,7 @@
 		<flag name='extras'>Enable Psi+ patcheset</flag>
 		<flag name='hunspell'>Enable spellchecking using <pkg>app-text/hunspell</pkg></flag>
 		<flag name='iconsets'>Install additional iconsets, some of them has not clear licensing</flag>
```

Comment:
..."some of them have no clear licensing"

Reply:
fixed